.github/workflows/push.yml

```diff
@@ -4,26 +4,21 @@ jobs:
   setup:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
-    - uses: actions/setup-node@master
-      with:
-        node-version: '10.x'
-    - name: Version
-      uses: ./.github/actions/version
-    - run: npm install
-    - name: Export SVG from Figma
-      uses: primer/figma-action@d7844a1927da9b8dd562cbf008c7da20e228607d
-      env:
-        FIGMA_FILE_URL: https://www.figma.com/file/FP7lqd1V00LUaT5zvdklkkZr/Octicons
-        FIGMA_TOKEN: ${{ secrets.FIGMA_TOKEN }}
-      with:
-        args: format=svg dir=./lib/build
-    - run: npm run lint
-    - run: npm test
-    - uses: actions/upload-artifact@master
-      with:
-        name: octicons
-        path: ./lib/build
+      - uses: actions/checkout@master
+      - uses: actions/setup-node@master
+        with:
+          node-version: '10.x'
+      - name: Version
+        uses: ./.github/actions/version
+      - run: npm install
+      - run: npm run build
+      - run: cp -r icons lib/build/svg
```

Comment:
Is it necessary to copy the icons into the build folder anymore? If we are reading from data.json in all the packages we might as well keep the folder out. Probably save in package size.

Reply:
I did this to pass tests but I agree that it seems unnecessary. However, removing the `svg` directory would technically be a breaking change which I was trying to avoid in this PR. Maybe we can remove the `svg` directory as part of the Octicons v2 release since that will also require a major version bump. How does that sound?